server/store.go

```diff
@@ -205,3 +206,66 @@ func (p *Plugin) removeSuperUserToken() error {
 
 	return nil
 }
+
+func (p *Plugin) storeUserPreference(userID, value string) error {
+	encoded, err := json.Marshal(value)
+	if err != nil {
+		return err
+	}
+
+	if err := p.API.KVSet(fmt.Sprintf(zoomUserPreferenceKey, userID), encoded); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (p *Plugin) getUserPreference(userID string) (string, error) {
+	encoded, err := p.API.KVGet(fmt.Sprintf(zoomUserPreferenceKey, userID))
+	if err != nil {
+		return "", err
+	}
+
+	if len(encoded) == 0 {
+		/*
+			If preference is not stored in kv store, check in the preferences table for user preference
+		*/
+		preferences, reqErr := p.API.GetPreferencesForUser(userID)
```

Comment:
Pleas use p.client here and everywhere else.

Reply:
Updated for rest. `GetPreferencesForUser` is not present in the user service.